services/market/tests/functional/market.prices.test.js

```diff
@@ -14,7 +14,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-/* eslint-disable no-console,no-multi-spaces,key-spacing,no-unused-vars */
+/* eslint-disable no-console */
```

Comment:
is this still needed `/* eslint-disable no-console */`?

Reply:
> is this still needed `/* eslint-disable no-console */`?

@nagdahimanshu Yes, since the broker instance sets the logger to console:
```
const broker = new ServiceBroker({	
	 transporter: 'redis://localhost:6379/0',	
	 logLevel: 'info',
	 requestTimeout: 15 * 1000,
	 logger: console,
});
```